projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.ts

```diff
@@ -82,12 +82,13 @@ export class AgentsButtonComponent implements OnInit, OnDestroy {
       .subscribe((selection) => {
         this.selectedNodesState = selection;
       });
+
     this.agentService
       .getAgents()
       .pipe(takeUntil(this.onDestroy$))
       .subscribe(
-        (paging) => {
-          this._agents = paging.list.entries.map((agentEntry) => agentEntry.entry);
+        (agents: AgentWithAvatar[]) => {
```

Comment:
we try to not set type in case when type can be auto detected. I think here it will be detected based on result of getAgents function from service